src/components/Card/Card.js

```diff
@@ -36,10 +36,15 @@ const channelsList = channels => {
 };
 
 const Avatar = ({ mentor }) => {
+  const mentorHtmlId = mentor.name.replace(/\s/g, '-');
```

Comment:
Can be extracting it to a function and reuse it in https://github.com/Coding-Coach/find-a-mentor/pull/230/files#diff-d893af3db783fc219af43033d801ec9dR64

Reply:
@moshfeu update done can you check if it is okei?